src/sgame/sg_bot_util.cpp

```diff
@@ -1382,12 +1408,13 @@ int FindBots( int *botEntityNumbers, int maxBots, team_t team )
 	int numBots = 0;
 	int i;
 	memset( botEntityNumbers, 0, sizeof( int )*maxBots );
-	for ( i = 0; i < MAX_CLIENTS; i++ )
+	for ( i = 0; i < MAX_CLIENTS && numBots < maxBots; i++ )
 	{
 		testEntity = &g_entities[i];
+		// testEntity->r.svFlags => "SVF_NOCLIENT, SVF_BROADCAST, etc..." good thing grep exists!
 		if ( testEntity->r.svFlags & SVF_BOT )
 		{
-			if ( testEntity->client->pers.team == team && numBots < maxBots )
+			if ( testEntity->client->pers.team == team )
```

Comment:
could use `G_Team()` too

Reply:
Ideally, I'd use a G_IsOnSameTeam() or alike that is pending in another branch, maybe I'll see if I can cherry pick it.